middlewares/validators/discordactions.js

```diff
@@ -28,7 +28,22 @@ const validateMemberRoleBody = async (req, res, next) => {
   }
 };
 
+const validateUpdateUsersNicknameStatusBody = async (req, res, next) => {
```

Comment:
improve this code block on below points

1. Use a cache: You can cache the schema object to avoid having to recompile it on every request.
2. Use a middleware: You can create a middleware function that validates the request body using the schema object. This will allow you to reuse the validation code in multiple routes.
3. Use asynchronous validation: You can use the validateAsync() method to validate the request body asynchronously. This will improve the performance of your application.

Reply:
1. Are you referring to caching the nicknames of the users that are being recompiled? Or caching the result of the request? 
2. I have split the validation code and the validator. Please confirm if this is what was expected.
3. I did use `validateAsync()`. Am I missing something here?